spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+RSpec.describe NotificationMailer, type: :mailer do
+  before do
+    workdey_data = Seed.new
+    workdey_data.create_all
+  end
+  describe "send_notifications" do
+    let(:user) { User.first  }
+    let(:task) { Task.last }
+    let(:task_category) { Task.first }
+    let(:notification_taskee) { User.first }
+    let(:notification_tasker) { User.last }
+    let(:mail) { NotificationMailer.send_notifications(user, 
+								    									task, 
+								    									task_category, 
+								    									notification_taskee, 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.<br>Trailing whitespace detected.